### PR TITLE
fix 14 and 15: define upstream + pull --rebase upstream

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: releaser/make_release.py [hunk 1]
```diff
@@ -409,7 +420,8 @@ def run_steps(config, steps, steps_funcs):
             step_func(config)
 
 
-def make_release(local_repository, package_name, module_name, release_name='dev', steps=':', branch='master',
-                 src_documentation=None, tmp_dir=None):
-    config = set_config(local_repository, package_name, module_name, release_name, branch, src_documentation, tmp_dir)
+def make_release(local_repository, project_name, package_name, module_name, release_name='dev', steps=':',
```
Comment (gdementen) [inline releaser/make_release.py]: Don't forget to update the "make_release.py" script of all projects accordingly.

**Review by gdementen — APPROVED** [review]
LGTM

**Review by gdementen — COMMENTED** [review]
wait a sec.

File: releaser/make_release.py [hunk 2]
```diff
@@ -289,6 +294,10 @@ def push(config):
         return
 
     chdir(config['repository'])
+    # rebase on upstream
+    doechocall('rebase on upstream before pushing main repository changes to GitHub',
```
Comment (gdementen) [inline releaser/make_release.py]: This probably does not hurt to do it here too, but this does NOT fix #15, as this rebases/push is for the "main" repository, not the feedstock repository, which is handled at: https://github.com/larray-project/releaser/blob/master/releaser/update_feedstock.py#L51

**Comment by gdementen** [comment]
@alixdamman what's the status of this PR?

**Comment by alixdamman** [comment]
This would change your way of using the "releaser", so I didn't wanted to push this PR. 
Maybe, I should closing it?

**Comment by gdementen** [comment]
Sorry, I don't remember what this was all about... In what way would it change the way we use the releaser? It's not necessarily a bad thing if the overall experience is better...

**Comment by alixdamman** [comment]
The releaser modify your local copy of larray, larray-editor, ...
When I started this PR, I thought the idea of the release was to clone the github repository in a temporary directory and to do everything from it until the release is done.

**Pull request closed without merging**